### ISSUE TITLE
Dragging in a CSV to generate a graph works in wysiwyg but not in markdown

### ISSUE THREAD

**Issue by jywarren** [publiclab/PublicLab.Editor#142]
in markdown, it is treated as a simple attachment -- the new `mergeHtmlAndAttachment: function (chunks, link) {` method is not invoked. 

https://github.com/publiclab/PublicLab.Editor/pull/137/files#diff-a25f5d4b5c50e14d0bd66616d381892aR22318

Need to look through woofmark to see when it's triggered and if we can intercept an attachment insertion in wysiwyg mode too...

**Comment by jywarren** [comment]
```
In wysiwyg mode:
[graph:https://publiclab.org/system/images/photos/000/024/807/original/114634.csv]

In markdown mode:
  [attachment-1]: https://publiclab.org/system/images/photos/000/024/808/original/114634.csv "114634.csv"
```

Example csv to use: https://publiclab.org/system/images/photos/000/024/808/original/114634.csv

**Comment by jywarren** [comment]
Relevant code segment:

https://github.com/publiclab/PublicLab.Editor/blob/f5f5492e5188963b0d6b672fc60093cef34c065c/src/adapters/PublicLab.Woofmark.js#L140-L151

This seems to work when you /convert/ between markdown and woofmark, but not when you initially drag it in. Odd!

**Comment by jywarren** [comment]
Working this over here: https://github.com/bevacqua/woofmark/issues/67

**Comment by sidntrivedi012** [comment]
@jywarren The output we need in markdown is getting wrongly parsed in the editor. It seems right in the code. Attaching image.
![screenshot from 2019-02-02 13-21-34](https://user-images.githubusercontent.com/31186013/52161623-dab65d80-26ed-11e9-8bdf-5fd0f51eefa6.png)

**Comment by jywarren** [comment]
Yes it seems we need to work on this upstream in the woofmark code, perhaps: https://github.com/bevacqua/woofmark/issues/67

**Comment by sidntrivedi012** [comment]
@jywarren Exactly. Should I make a issue there regarding this?

**Comment by rexagod** [comment]
A workaround for this could be using `crossvent` to check if there's any `.csv` link present in the `wk-wysiwyg` and converting that to a powertag. How does that sound?

**Comment by jywarren** [comment]
I think we can try to work it both ways... @sidntrivedi012 take a look at this, there was a suggestion for how to do it: https://github.com/bevacqua/woofmark/issues/67#issuecomment-386880691

And @rexagod would you be willing to try an implementation? Could we write a test against it to ensure it works?

**Comment by sidntrivedi012** [comment]
Sure @jywarren. Working on it already.

**Comment by rexagod** [comment]
I experimented a bit with `crossvent`( now '.csv' links in markdown are automatically converted to `'<div class="powertags">Power tag: graph:' + href.link + '</div>'`) but I still think we can go further and replace the injected HTML by some fancy characters like we did in `AutoCenter` with -> and <- so as to make this more readable and easily operable. How does that sound?

![peek 2019-02-08 20-09](https://user-images.githubusercontent.com/33557095/52489157-875d7700-2be7-11e9-96cd-c7c00b34ce7f.gif)

**Comment by jywarren** [comment]
That sounds great! Do you want to implement the quick fix and open an issue
for a more advanced version, or what do you think?

Thanks so much for looking at this!
